src/hooks/useHoldings.ts

```diff
@@ -66,7 +66,9 @@ export default function useHoldings(): {
   const data: HoldingsApi = { ...celoHoldings.data, ...otherHoldings.data };
 
   const isLoadingCelo = useMemo(() => {
-    return data?.celo?.frozen?.updated === 0 || data?.celo?.unfrozen?.updated === 0;
+    return (
+      data?.celo?.frozen?.updated === 0 || data?.celo?.unfrozen?.updated === 0
```

Comment:
Could potentially pop a toast that certain data endpoint is down, don't know if we want optics on that though

Reply:
Ideally, we should show cached data and let people know that it's not the most recent. This behavior would be good to incorporate into the more significant overhaul of the reserve site.